config/database.go

```diff
@@ -293,7 +293,13 @@ func (ds *DatabaseStore) RemoveFile(name string) error {
 
 // String returns the path to the database backing the config, masking the password.
 func (ds *DatabaseStore) String() string {
-	u, _ := url.Parse(ds.originalDsn)
+	tcpStripper := regexp.MustCompile(`@tcp\((.*)\)`)
+
+	// Remove @tcp and the branches from the host and parse the rest as URl
+	u, err := url.Parse(tcpStripper.ReplaceAllString(ds.originalDsn, `@$1`))
+	if err != nil {
+		return ""
```

Comment:
Suggest we return something explicit here:
```suggestion
		return "(omitted due to error parsing)"
```